.circleci/config.yml

```diff
@@ -230,12 +230,16 @@ jobs:
     steps:
       - attach_workspace:
           at: /tmp/workspace
+      - checkout
+      - run: |
+          # CRYSTAL_NEXT_VERSION uses src/VERSION to publish maintenance releases as prereleases
+          echo "export CRYSTAL_NEXT_VERSION=$(cat src/VERSION)" >> /tmp/workspace/distribution-scripts/build.env
       - run: |
           cd /tmp/workspace/distribution-scripts
 
           # How to brand it
           export VERSION=${CIRCLE_BRANCH/release\//}
-          export VERSION=${VERSION/\//-}-dev
+          export VERSION=${VERSION//\//-}-dev
```

Comment:
was something wrongly replaced?

Reply:
If the branch name has more than one `/`, the previous expression only replaced the first one. The remaining ones would cause an error for invalid package version.